indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -118,6 +129,76 @@ class BookingDetails extends React.Component {
     );
   };
 
+  renderPreBookingWarning = () => {
+    const {bookingStateChangeInProgress} = this.props;
+    const {
+      acceptanceData,
+      actionInProgress,
+      preBookingWarningVisible,
+      concurrentBookings,
+    } = this.state;
+    const items = concurrentBookings.map(booking => {
+      const reservation = booking.reservation;
+      const startDate = serializeDate(toMoment(booking.start_dt), 'L LT');
+      const endDate = serializeDate(toMoment(booking.end_dt), 'LT');
+      return (
+        <List.Item key={reservation.id}>
+          <a
+            href={bookingLinkURL({booking_id: reservation.id})}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {startDate} - {endDate}
+          </a>
+          <Translate>
+            {' '}
+            by <Param name="user" value={reservation.booked_for_name} /> {'('}
+            <Param name="info" value={reservation.booking_reason} />
+            {')'}
+          </Translate>
+        </List.Item>
+      );
+    });
+    return (
+      <Modal open={preBookingWarningVisible} size="small" onClose={this.hidePreBookingWarning}>
+        <Modal.Header>
+          <Translate>Warning</Translate>
+        </Modal.Header>
+        <Modal.Content>
+          <Translate>
+            There is more than one booking request during this time. Accepting this booking will
+            automatically reject the following:
+          </Translate>
+          {!!concurrentBookings.length && (
```

Comment:
Is this check needd? I wouldn't expect this warning to show up at all unless there *are* conflicting prebookings ;)